addons/mail/models/mail_activity.py

```diff
@@ -435,22 +439,34 @@ def write(self, values):
                 self.env[activity.res_model].browse(activity.res_id).message_subscribe(partner_ids=[activity.user_id.partner_id.id])
                 if activity.date_deadline <= fields.Date.today():
                     self.env['bus.bus'].sendone(
-                        (self._cr.dbname, 'res.partner', activity.user_id.partner_id.id),
-                        {'type': 'activity_updated', 'activity_created': True})
+                        activity.user_id.partner_id,
+                        'mail.activity_updated',
+                        {
+                            'activity_created': True,
```

Comment:
payload change: I think this key is useless, `activity_created` should be the type, and no payload.